README.md

```diff
@@ -1 +1,25 @@
-# backend-hono
\ No newline at end of file
+# BACKEND-HONO
+
+### STACK:
+
+- Framework - hono
+- Orm - Prisma ORM
+- Database - SQLite3
+- TS
+- Prettier/Eslint
+- Postman
```

Comment:
It’s for you to test what you wrote. It’s not a part of the stack. You can remove it)